js/eventListener.js

```diff
@@ -6,26 +6,26 @@ let guling = document.querySelectorAll(".pots");
 
 //let keys = document.querySelectorAll(".key");
 
-let playlist = ["", "./assets/gulingtanga-notes/Db4.wav", "./assets/gulingtanga-notes/Eb4.wav", "", "./assets/gulingtanga-notes/Gb4.wav", "./assets/gulingtanga-notes/Ab4.wav", "./assets/gulingtanga-notes/Bb4.wav", "./assets/gulingtanga-notes/C4", "./assets/gulingtanga-notes/D4", "./assets/gulingtanga-notes/E4", "./assets/gulingtanga-notes/F4", "./assets/gulingtanga-notes/G4", "./assets/gulingtanga-notes/A5", "./assets/gulingtanga-notes/B5", "./assets/gulingtanga-notes/C5"];
+let notes=document.querySelectorAll(".notes");
```

Comment:
Try ".pots:not(.empty)"

This will select all the pots class div which is not with the empty class